libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1267,19 +1267,25 @@ namespace SohImGui {
                     ImGui::PopStyleVar(3);
                     ImGui::PopStyleColor(1);
 
-                    // TODO mutual exclusions -- There should be some system to prevent conclifting enhancements from being selected
-                    PaddedEnhancementCheckbox("D-pad Support on Pause and File Select", "gDpadPauseName");
-                    Tooltip("Enables Pause and File Select screen navigation with the D-pad\nIf used with D-pad as Equip Items, you must hold C-Up to equip instead of navigate");
+                    PaddedEnhancementCheckbox("D-pad Support on Pause and File Select", "gDpadPauseName", true, false);
+                    Tooltip("Navigate Pause and File Select with the D-pad\nIf used with D-pad as Equip Items, you must hold C-Up to equip instead of navigate");
                     PaddedEnhancementCheckbox("D-pad Support in Text Choice", "gDpadText", true, false);
                     PaddedEnhancementCheckbox("D-pad Support for Browsing Shop Items", "gDpadShop", true, false);
                     PaddedEnhancementCheckbox("D-pad as Equip Items", "gDpadEquips", true, false);
-                    Tooltip("Allows the D-pad to be used as extra C buttons");
+                    Tooltip("Equip items and equipment on the D-pad\nIf used with D-pad on Pause and File Select, you must hold C-Up to equip instead of navigate");
```

Comment:
this feels like an unrelated change, i'm all for it in this case, but in general it's nice to keep PRs on topic

Reply:
Fair.  I actually want to consolidate some of those D-pad cvars anyway now that the D-pad Ocarina one has been moved out of the main menu, so I'll revert these tonight!